vendor.conf

```diff
@@ -67,9 +64,13 @@ github.com/theupdateframework/notary                bf96a202a09a312ae005cd312fc0
 github.com/tonistiigi/fsutil                        d72af97c0eaf93c1d20360e3cb9c63c223675b83
 github.com/tonistiigi/go-rosetta                    f79598599c5d34ea253b56a1d7c89bc6a96de7db
 github.com/tonistiigi/units                         6950e57a87eaf136bbe44ef2ec8e75b9e3569de2
+github.com/tonistiigi/vt100                         8066bb97264f56c603341d3fce837bb504e662ef # v0.0.0-20210615222946-8066bb97264f
 github.com/xeipuuv/gojsonpointer                    02993c407bfbf5f6dae44c4f4b1cf6a39b5fc5bb
 github.com/xeipuuv/gojsonreference                  bd5ef7bd5415a7ac448318e64f11a24cd21e594b
 github.com/xeipuuv/gojsonschema                     82fcdeb203eb6ab2a67d0a623d9c19e5e5a64927 # v1.2.0
+go.opentelemetry.io/contrib                         91e5180ae03828601f705ac8e7fc3bd9e34dae6a # v0.21.0
```

Comment:
We need to check if we need the fork for this; https://github.com/moby/buildkit/blob/v0.9.0/go.mod#L79-L83

Reply:
Yes indeed I didn't take these ones into account. I think it was temporary and should be fixed upstream soon hopefully:
* open-telemetry/opentelemetry-go-contrib#873
* open-telemetry/opentelemetry-go-contrib#874
* open-telemetry/opentelemetry-go-contrib#875

Reply:
Added a commit to use that fork